adagrams/game.py

```diff
@@ -1,11 +1,130 @@
+from operator import le
+import string
+import random
+import copy
+
 def draw_letters():
-    pass
+    
+    pool_of_letters_dict = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+    string.ascii_letters
```

Comment:
Seems this is a literal standing alone on its own line; since there is no statement doing anything to/with it, feel free to delete it.